README.md

```diff
@@ -52,28 +52,34 @@ Eventually, you should pass your token in the `Authorization` header for the rel
 
 ## Development
 
-The project is using [Stack](https://docs.haskellstack.org/en/stable/README/).
+The project provides scripts to manage the lifecycle of the application. If `nix` is available the scripts will attempt to use it, otherwise they will fall back to the "raw" commands (i.e., using `stack` directly from the user's environment.)
```

Comment:
I think it would make sense to provide the three options to the user:
- use `stack` directly
- use `nix` directly
- use our wrapper scripts
We could phrase it something like `The project is setup to be built either with Stack or Nix. We provide also convenience scripts which try to use Nix, if available, and fall back to using Stack directly otherwise`.

Reply:
Another alternative is to use a flag in the script to decide whether to use `nix` or `stack` directly